### ISSUE TITLE
On-chain graph data CLI and REST query method

### ISSUE THREAD

**Issue by aaronc** [regen-network/regen-ledger#26]
For `xrncli query data get` and `/data/<bech32-url-encoded>`

Blocked by [#21](https://github.com/regen-network/regen-ledger/issues/21) (needs an off-chain `SchemaResolver`)

For the scope of this ticket it should be possible to retrieve data in binary format at a minimum. It would also be desirable to return it in JSON-LD with proper support for framing but that will be addressed in another ticket.

**Comment by clevinson** [comment]
@aaronc should this get closed ?
